.drone.star

```diff
@@ -658,12 +658,14 @@ def changelog(ctx):
         },
     }]
 
-# replaces reserved and unsafe url characters with '-'
-# reserved: & $ + , / : ; = ? @ #
-# unsafe: <space> < > [ ] { } | \ ^ %
-def sanitizeBranchName(BRANCH_NAME = ""):
+def setPactConsumerTagEnv(ctx):
+    consumer_tag = ctx.build.source if ctx.build.event == "pull_request" else ctx.repo.name
```

Comment:
The PR description says "...and for push/tags events the tag name will be the repo default branch name (master)."

But I think that `ctx.repo.name` os going to be something like `owncloud/owncloud-sdk` or `owncloud-sdk`

The PR description does not seem to match the code.

Reply:
Ohh yeah! I have completely mistaken here. It should be the repo's default branch.